README.md

```diff
@@ -58,7 +58,7 @@ We recommend addressing all the issues that Error Prone reports, particularly th
 
 Versions 3.0.0 and later of the Gradle Error Prone Plugin [no longer support Android](https://github.com/tbroyer/gradle-errorprone-plugin/releases/tag/v3.0.0).  So if you're using a recent version of this plugin, you'll need to add some further configuration to run Error Prone and NullAway.  Our [sample app `build.gradle` file](https://github.com/uber/NullAway/blob/master/sample-app/build.gradle) shows one way to do this, but your Android project may require tweaks.  Alternately, 2.x versions of the Gradle Error Prone Plugin still support Android and may still work with your project.
 
-Beyond that, compared to the Java configuration, the `com.google.code.findbugs:jsr305:3.0.2` dependency can be removed; you can use the `android.support.annotation.Nullable` annotation from the Android Support library instead.
+Beyond that, compared to the Java configuration, the JSpecify dependency can be removed; you can use the `android.support.annotation.Nullable` annotation from the Android Support library instead.
```

Comment:
@cpovirk are we currently encouraging Android users to switch over from Android support annotations to JSpecify as well?  Or will that cause problems for them?

Reply:
Phrasing is difficult :) We have had similar conversations internally.

On the one hand, it would be nice if we could find a way to avoid implying that the Android annotations (which of course are declaration annotations) are "just as good" as the JSpecify ones.

On the other hand, it would be nice to make clear there there are still tradeoffs, by which I mainly mean "Dagger (if you don't use a recent javac)." (On the plus side, maybe Android users can more upgrade their compiler especially easily, since they don't need to worry about setting `--release`? Part of me now wants for the NullAway docs to demonstrate configuring a Java 22 toolchain for Android, except that it's not really fair to put that on the critical path for NullAway adopters :))

I was hoping to convince myself that we had an existing doc that covered this, but as you saw, I ended up leaving [a comment on a JSpecify issues](https://github.com/jspecify/jspecify/issues/550#issuecomment-2240050575) instead.

The best link at this point is probably https://jspecify.dev/docs/whether/#annotation-processors. Let me see if I can think of a way that I'd phrase that once I get back to my desk after a quick break. I'm hoping to propose a different framing for this sentence.

Reply:
On second thought: I might be putting too much emphasis on the ability to annotate type arguments, type-parameter bounds, and array-element types yet, since NullAway support there is still developing. (And of course NullAway already offers a way to treat unannotated types as non-nullable when you want it, so there's less reason for `@NullMarked`, too.)

I think I'd probably just keep this as you have it for now, and we can revisit once we have docs that speak more directly to the tradeoffs. Maybe we'll also tweak the comment in the code block at line 29 at that point.

---

Hmm, but now that I'm here: Nowadays, people use `androidx.annotation.Nullable` instead of `android.support.annotation.Nullable`, right?

Reply:
Will switch to refer to `androidx`, thanks!